helm/values.yaml

```diff
@@ -180,3 +204,81 @@ cluster:
 # Leave it empty if not needed
 proxy:
   no_proxy:
+
```

Comment:
That's a good point, but I'm not sure if it's really worth creating a jsonschema for it instead of specifying it somewhere (e.g. in the PR). At least I wouldn't include the jsonschema in this PR and make it a topic, as it would again take some time and is not really the goal of this PR.